lib/dlss/capistrano/tasks/check_status.rake

```diff
@@ -0,0 +1,15 @@
+# Capistrano plugin hook to set default values
+namespace :load do
+  task :defaults do
+    set :check_status_roles, fetch(:check_status_roles, [:web])
+    set :check_status_path, fetch(:check_status_path, '/status/all')
+  end
+end
+
+desc 'Run status checks'
+task :check_status do
+  on roles fetch(:check_status_roles) do |host|
+    info "Checking status at https://#{host}#{fetch(:check_status_path)}:"
+    puts capture("curl https://#{host}#{fetch(:check_status_path)}")
```

Comment:
Another way to do this would be:
```
 with rails_env: :production do
   runner "puts URI.open('https://example.com').read"
 end
```

Then we wouldn't have a `curl` dependency. Of course `curl` is likely faster 🤷

Reply:
@jcoyne and I think our server boxes tend to have `curl`